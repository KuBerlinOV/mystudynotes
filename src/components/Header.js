import React from 'react';
import { NavLink } from 'react-router-dom';


const Header = () => (
    <header>
        <NavLink activeClassName='is-active' className="nav-link" to="/" exact={true}>Home</NavLink>
        <NavLink activeClassName='is-active' className="nav-link" to="/notes">Notes</NavLink>
        <NavLink activeClassName='is-active' className="nav-link" to="/about">Organizing your studies</NavLink>
    </header>
)

export default Header;
