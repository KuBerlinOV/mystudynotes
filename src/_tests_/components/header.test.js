// import ReactShallowRenderer from 'react-test-renderer/shallow';

import React from 'react';
import Header from '../../components/Header';
import { shallow } from 'enzyme';



// test('should render header correctly', () => {
//     const renderer = new ReactShallowRenderer();
//     renderer.render(<Header />);
//     expect(renderer.getRenderOutput()).toMatchSnapshot();

//     console.log(renderer.getRenderOutput());
// })

describe('Header', () => {
    it('should render Header correctly', () => {
        const wrapper = shallow(<Header />)
        expect(wrapper).toMatchSnapshot();
    })
})
