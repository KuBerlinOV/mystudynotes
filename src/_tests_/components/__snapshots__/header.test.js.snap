// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render Header correctly 1`] = `
<header>
  <NavLink
    activeClassName="is-active"
    className="nav-link"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    className="nav-link"
    to="/notes"
  >
    Notes
  </NavLink>
  <NavLink
    activeClassName="is-active"
    className="nav-link"
    to="/about"
  >
    Organizing your studies
  </NavLink>
</header>
`;

exports[`should render Header correctly 1`] = `ShallowWrapper {}`;

exports[`should render header correctly 1`] = `
<header>
  <NavLink
    activeClassName="is-active"
    className="nav-link"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    className="nav-link"
    to="/notes"
  >
    Notes
  </NavLink>
  <NavLink
    activeClassName="is-active"
    className="nav-link"
    to="/about"
  >
    Organizing your studies
  </NavLink>
</header>
`;
